src/components/Navigator/NavigatorCard.vue

```diff
@@ -480,6 +483,14 @@ export default {
       // if we remove APIChanges, remove all related tags as well
       this.selectedTags = this.selectedTags.filter(t => !ChangeNames[t]);
     },
+    async activeUID(newUid, oldUID) {
+      // update the dynamicScroller item's size, when we change the UID
```

Comment:
Can you also mention the reason we need to do this, which is because the size may change due to styling we apply to the active item?